sql/core/src/test/resources/sql-tests/results/predicate-functions.sql.out

```diff
@@ -263,7 +263,7 @@ select to_date('2017-03-01') = to_timestamp('2017-03-01 00:00:00')
 -- !query 32 schema
 struct<(CAST(to_date('2017-03-01') AS TIMESTAMP) = to_timestamp('2017-03-01 00:00:00')):boolean>
 -- !query 32 output
-true
+false
```

Comment:
This could look weird but `'2017-03-01'` is in UTC time zone, `2017-03-01 00:00:00` is parsed in session local time zone (`Los Angeles`). We could point out the UTC time zone explicitly, for example: `to_timestamp('2017-03-01 00:00:00Z')`

Reply:
shall we change the test case instead?

Reply:
Sure